_posts/2020-08-26-engineering-firstweek.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "My first week in Scribd engineering"
+tags:
+- featured
+team: Core Infrastructure
+author: zherner
```

Comment:
```suggestion
author: jasond
```

Just noticed this too, for future reference the author key should be what's on line 97  from authors.yml